### ISSUE TITLE
Compile Issue error: cannot find symbol

### ISSUE THREAD

**Issue by alisabry22** [khjde1207/tesseract_ocr#46]
I jave an issue that prevents me from the app starting .

using latest version in pupspec.yaml:
```
dependencies:
  flutter_tesseract_ocr: ^0.4.25
```
also defined tessdata folder and files included in it , with tessdata_config.json
Issue :
/Users/alisabry/.pub-cache/hosted/pub.dev/flutter_tesseract_ocr-0.4.25/android/src/main/java/io/paratoner/flutter_tesseract_ocr/FlutterTesseractOcrPlugin.java:51: error: cannot find symbol

full error stack:
```
/Users/alisabry/.pub-cache/hosted/pub.dev/flutter_tesseract_ocr-0.4.25/android/src/main/java/io/paratoner/flutter_tesseract_ocr/FlutterTesseractOcrPlugin.java:51: error: cannot find symbol
        final Map<String, String> args = call.argument("args");
              ^
  symbol:   class Map
  location: class FlutterTesseractOcrPlugin
/Users/alisabry/.pub-cache/hosted/pub.dev/flutter_tesseract_ocr-0.4.25/android/src/main/java/io/paratoner/flutter_tesseract_ocr/FlutterTesseractOcrPlugin.java:65: error: package Map does not exist
  for (Map.Entry<String, String> entry : args.entrySet()) {
                  ^
/Users/alisabry/.pub-cache/hosted/pub.dev/flutter_tesseract_ocr-0.4.25/android/src/main/java/io/paratoner/flutter_tesseract_ocr/FlutterTesseractOcrPlugin.java:105: error: cannot find symbol
      baseApi.end();
```
* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 38s
Error: Gradle task assembleDebug failed with exit code 1

**Comment by ErsinMandaci** [comment]
I have the same problem, what is the solution?

**Comment by khjde1207** [comment]
Fixed in version 0.4.26.

**Comment by alisabry22** [comment]
@khjde1207  highly appreciated .. keep good work up.
